app/components/Application/ApplicationDetailsCardItemCustomFields.tsx

```diff
@@ -75,6 +76,13 @@ const customFields = (showDiff, diffPathArray, diffArray, handleEnums) => {
     emissionGas: props => <SummaryEmissionGasFields {...props} />,
     production: props => (
       <ProductionFields query={props.formContext.query} {...props} />
+    ),
+    NumberField: props => (
+      <NumberFormat
+        thousandSeparator
+        displayType="text"
+        value={props.formData}
+      />
```

Comment:
This needs to handle the display of differences, the same way `StringField` does

Reply:
Updated.